app/src/main/java/com/example/next_show/models/Show.java

```diff
@@ -127,6 +137,18 @@ private static boolean isForbiddenShow(String showID) {
         return forbiddenShows.contains(showID);
     }
 
+    private static boolean isSaved(String slug) {
+        User user = (User) ParseUser.getCurrentUser();
+        List<String> savedShows = user.getAllSavedShows();
+
+        // check if user has saved shows -> trivially true
+        if (savedShows == null || savedShows.isEmpty()) {
+            return true;
+        }
```

Comment:
Hm, should this be... false?

Reply:
Yes! Thank you for catching it, fixed for the next commit :)